Makefile

```diff
@@ -36,6 +36,7 @@ install:
 .PHONY: build-core
 build-core:
 	$(MAKE) -C semgrep-core
+	$(MAKE) -C semgrep-core install
```

Comment:
Is this accidental?

Reply:
No, I wanted to make it so that `build-core` would be equivalent to `make && make install`. Happy to do something different! The goal is to make it so that people can set up `semgrep-core` without `semgrep` in two commands (`make dev-setup` and `build-core` right now). I also considered adding a separate setup command that combined `make dev-setup` and the `make && make install`. Let me know what you think I should do